### PR TITLE
beef up manifest and link to it only once

### PR DESCRIPTION
**Pull request by chrisburnell**
Add a number of extra fields to the manifest file, and ensure it is linked to only once in the markup.

### CODE REVIEW

File: static/assets/manifest.json [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {
-    "name": "SOTB 6",
+    "manifest_version": 2,
+    "dir": "ltr",
+    "lang": "en",
+    "default_locale": "en",
+    "short_name": "SotB18",
+    "name": "State of the Browser 2018",
+    "description": "State of the Browser is a one-day, single-track conference with widely varying talks about the modern web",,
```
Comment (ThePeach) [inline static/assets/manifest.json]: double comma? I guess a typo

File: static/assets/manifest.json [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+    "display": "standalone",
+    "orientation": "any",
+    "start_url": "/",
+    "homepage_url": "https://2018.stateofthebrowser.com/",
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
```
Comment (ThePeach) [inline static/assets/manifest.json]: I would use `#f7f5f5` as a colour

File: templates/partials/page-favicons.hbs [hunk 3]
```diff
@@ -1,6 +1,5 @@
 <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
 <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.ico">
 <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.ico">
-<link rel="manifest" href="/manifest.json">
```
Comment (ThePeach) [inline templates/partials/page-favicons.hbs]: any reason for removing it from the HTML ?

Comment (chrisburnell) [reply templates/partials/page-favicons.hbs]: Already linked in [templates/partials/page-head.hbs](https://github.com/londonwebstandards/sotb-website/blob/manifest-fix/templates/partials/page-head.hbs#L8)

Comment (ThePeach) [reply templates/partials/page-favicons.hbs]: perfect! missed it

File: static/assets/manifest.json [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+    "default_locale": "en",
+    "short_name": "SotB18",
+    "name": "State of the Browser 2018",
+    "description": "State of the Browser is a one-day, single-track conference with widely varying talks about the modern web",
+    "display": "standalone",
+    "orientation": "any",
+    "start_url": "/",
+    "homepage_url": "https://2018.stateofthebrowser.com/",
+    "theme_color": "#f5f5f5",
+    "background_color": "#f5f5f5",
```
Comment (ThePeach) [inline static/assets/manifest.json]: `#f7f5f5`
it's a slightly red-ish grey :)

**Pull request merged**